examples/README.md

```diff
@@ -3,13 +3,15 @@
 This folder contains examples showing how to use lettre in your own projects.
 
 ## Examples
+- [file_html.rs] - Create an HTML email and store it in a local directory using `FileTransport`.
```

Comment:
I wouldn't mention the storing part here. Most users are probably going to want to send it for real.

```suggestion
- [file_html.rs] - Create an HTML email.
```